doc/releases/changelog-dev.md

```diff
@@ -17,6 +17,9 @@
 
 * Upgrade Pauli arithmetic with subtraction `w1 - w2 - 1`.
   [(#5003)](https://github.com/PennyLaneAI/pennylane/pull/5003)
+
+* Upgrade Pauli arithmetic with multiplying by scalars, addition and subtraction. You can now construct operators like `0.5 * w1 - 1.5 * w2 + 2`.
+  [(#5017)](https://github.com/PennyLaneAI/pennylane/pull/5017)
```

Comment:
Often we leave it as a single changelog entry and just add more to it and add another line after each subsequent PR, I think that's what Christina had in mind. [Like this one](https://github.com/PennyLaneAI/pennylane/pull/4654/files).

Reply:
Aaaaaaah 🙈 will update that, makes more sense!

Reply:
Is this what you had in mind now? Since the diff looks a bit messy here a screenshot: 
<img width="1024" alt="image" src="https://github.com/PennyLaneAI/pennylane/assets/43949391/3e4df39d-6941-466f-b609-f507fc7cd46d">